addons/mail/static/src/components/attachment/attachment_tests.js

```diff
@@ -667,6 +667,94 @@ QUnit.test('[technical] does not crash when the viewer is closed before image lo
     }
 });
 
+QUnit.test('text attachment is viewable', async function (assert) {
+    assert.expect(1);
+
+    await this.start();
+    const attachment = this.env.models['mail.attachment'].create({
+        filename: "test.txt",
+        id: 750,
+        mimetype: 'text/plain',
+        name: "test.txt",
+    });
+    await this.createAttachmentComponent(attachment, {
+        detailsMode: 'card',
+        isDownloadable: false,
+        isEditable: false,
+    });
+    assert.hasClass(
+        document.querySelector('.o_Attachment_image'),
+        'o-attachment-viewable',
```

Comment:
Assert on classname modifier of the root node of component.

_(this comment applies to all tests)_